mysql-test/suite/rpl/t/rpl_update_partitioned_table_with_compressed_column.test

```diff
@@ -0,0 +1,18 @@
+#
+# PS-7599 : Node inconsistency seen in PS during update on compressed column table
+#
+--source include/have_binlog_format_row.inc
+--source include/master-slave.inc
+
+CREATE TABLE t1 (f1 INT, f2 BLOB COLUMN_FORMAT COMPRESSED) ENGINE=INNODB PARTITION BY HASH (f1) PARTITIONS 2;
+
+REPLACE INTO t1 (f1,f2)VALUES(0,'123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456');
+
+UPDATE t1 SET f1=1 WHERE f1 = 0;
+
+--source include/sync_slave_sql_with_master.inc
+--source include/rpl_connection_master.inc
```

Comment:
In between these, can we validate that both master and slave have same data?

We can just source diff_tables.inc
```
--let $diff_tables= master:t1,slave:t1
--source include/diff_tables.inc
```